exporting/read_config.c

```diff
@@ -389,7 +389,10 @@ struct engine *read_exporting_config()
 
             tmp_instance->config.connector_specific_config = connector_specific_config;
 
-            connector_specific_config->stream_name = strdupz(exporter_get(instance_name, "stream name", "netdata"));
+            connector_specific_config->stream_name = strdupz(exporter_get(instance_name, "stream name", ""));
+            if (!strcmp(connector_specific_config->stream_name, ""))
```

Comment:
I would check it in `init_aws_kinesis_instance()` and return 1 on error. Then the connector instance won't even start.

Reply:
All right, I changed the PR, but we will need to take care of the buffer in another PR as we talked before.